Assets/Scripts/DataTypes/Edges/Edge.cs

```diff
@@ -104,7 +119,7 @@ private void Display()
                 triangles = triangles.ToArray(),
                 uv = uvs.ToArray()
             };
-            var tiling = Mathf.RoundToInt(shape.length * DISTANCE_UNIT / LINE_LENGTH);
+            var tiling = Mathf.RoundToInt(shape.length.ToDistanceUnits() * DISTANCE_UNIT / LINE_LENGTH);
```

Comment:
Aus `.ToDistanceUnits() * DISTANCE_UNIT` `.ToMeters()` machen, wenn das möglich ist

Reply:
Jetzt bin ich verwirrt.
`.ToDistanceUnits()` macht schon `length.Meters * DISTANCE_UNIT`.
Warum machen wir das hier nochmal doppelt?

Reply:
Wir wollen die Länge in Metern, früher hat `length` die Länge in Distance-units gespeichert, deswegen mussten wir das mit `DISTANCE_UNIT` multiplizieren. Jetzt können wir direkt auf dir Länge in Metern zugreifen.

Reply:
Also wenn ich einfach das nutze, ist die ganze Straße komisch grau:
```cs
var tiling = Mathf.RoundToInt((float) shape.length.Meters / LINE_LENGTH);
```

Reply:
Dann schaue ich mir das nochmal an.